morf-core/src/main/java/org/alfasoftware/morf/sql/UseParallelDml.java

```diff
@@ -15,6 +16,23 @@
  */
 public class UseParallelDml implements Hint {
 
+  private final Integer degreeOfParallelism;

```

Comment:
If you are using `Optional<Integer>` for the return from `getDegreeOfParallelism()`,
then let's store the value as `Optional<Integer>` too.